Fix Travis build

pluradj opened the pull request:
* Include targets against ONNX official releases
* Allow failures against ONNX latest dev branch

Signed-off-by: Jason Plurad <pluradj@us.ibm.com>

chinhuang007 commented on test/backend/test_onnx_backend.py:
```diff
@@ -132,6 +132,22 @@ def skip_not_implemented_ops_test(test):
   # Do not support dilations != 1 for ConvTranspose, test is added in opset 10
   backend_test.exclude(r'[a-z,_]*convtranspose_dilations[a-z,_]*')
 
+# Concat from sequence with new_axis=1 not supported
+backend_test.exclude(r'test_sequence_model5_[a-z,_]*')
+
+# GreaterOrEqual
+backend_test.exclude(r'test_greater_equal_[a-z,_]*')
+# LessOrEqual
+backend_test.exclude(r'test_less_equal_[a-z,_]*')
+# NegativeLogLikelihoodLoss and SoftmaxCrossEntropyLoss
+backend_test.exclude(r'test_softmax_cross_entropy_[a-z,_]*')
+# Dropout version 12
+backend_test.exclude(r'test_training_dropout_[a-z,_]*')
+# fails tolerance
+backend_test.exclude(r'test_gru_seq_length_[a-z,_]*')
+# KeyError: ''
+backend_test.exclude(r'test_nesterov_momentum_[a-z,_]*')
```
The filtering here currently is purposed to skip the tests we know onnx-tf will not pass for known reasons and we have not plans in the future to enable the tests. It is not targeted for any particular ONNX opset version so a developer can run and see the failed cases against any ONNX version, such as 1.6, 1.7, or master.

Is it possible to separate the filtering for the tests that we wanted to skip all the time from the ones we plan to implement but not getting to them yet?

Also, if the goal is to pass CI, especially for the target ONNX version, maybe we need to introduce the ONNX version (opset version) to the code?

chinhuang007 commented:
@pluradj This looks really good. One more request, could you please also remove the files, setup_onnx_master.sh and setup_onnx_target.sh, that are no longer needed with this approach?
Since we are not using docker for CI anymore, I think we should clean up the inactive files to make our repo clean. Thanks!

chinhuang007 commented:
@pluradj Just saw a typo in https://github.com/onnx/onnx-tensorflow/blob/36a74f56f1f9dcd152bdb9319184f14c7551f179/onnx_tf/common/data_type.py#L104, unit64 should be uint64, caught by the CI. Could you please make the typo fix as well? Thanks!

pluradj commented:
> @pluradj Just saw a typo in
> 
> https://github.com/onnx/onnx-tensorflow/blob/36a74f56f1f9dcd152bdb9319184f14c7551f179/onnx_tf/common/data_type.py#L104
> , unit64 should be uint64, caught by the CI. Could you please make the typo fix as well? Thanks!

Updated, removed Docker pieces from Travis, and also added some backend exclusions for `pow` operator.

chinhuang007 commented:
Thanks a lot for getting CI fixed!
